Sources/xcodeproj/Project/XCBreakpointList.swift

```diff
@@ -401,7 +401,7 @@ public final class XCBreakpointList: Equatable, Writable {
 
     // MARK: - Writable
 
-    public func write(path: AbsolutePath, override: Bool) throws {
```

Comment:
I'd keep the `override` attribute and discuss first whether it makes sense to keep it in the public API or get rid of it.

Reply:
Do we know if anyone is using it?

Reply:
It's hard to say because we don't have a list of all the projects that are using the project. I'd keep it as it was an open an issue explaining why you think we should deprecate and get rid of the argument. If we end up doing it, it's better to do it in a separate PR so that we can track it down easily.